__tests__/app.test.js

```diff
@@ -85,3 +85,59 @@ describe("GET /api/categories", () => {
     });
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  describe("SUCESS", () => {
+    it("should respond with status code 200", () => {
+      return request(app).get("/api/reviews/1").expect(200);
+    });
+
+    it("should respond with the requested review object, containing all corresponding properties and values", () => {
+      return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then(({ body }) => {
+          const { review } = body;
+
+          expect(review).toMatchObject({
+            review_id: 1,
+            title: "Agricola",
+            category: "euro game",
+            designer: "Uwe Rosenberg",
+            owner: "mallionaire",
+            review_body: "Farmyard fun!",
+            review_img_url:
+              "https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700",
+            created_at: "2021-01-18T10:00:20.514Z",
+            votes: 1,
+          });
+
+          expect(review).toHaveProperty("review_id", 1);
+        });
```

Comment:
Really nice test here to be specific with the review you are expecting, nice job!